src/apps/components/run-header.js

```diff
@@ -54,15 +54,28 @@ class RunHeader extends React.Component {
 
         let query = !this.props.bundle ? { local: true } : this.props.location.query;
 
+        var codeBtn;
+        if (this.props.runMode.path || this.props.runMode.rendezvous || this.props.runMode.local) {
```

Comment:
This is a nit that you do or not:
Sometimes with three conditions like you have here it's hard to understand the intention. If you had a function like `hasRunMode` (or something even more descriptive of intention) it might make the code easier to understand.